src/components/table/TableCellExpander.tsx

```diff
@@ -0,0 +1,54 @@
+import { CellProps } from 'react-table';
+import { ReactComponent as IconChevronDown } from 'assets/icons/chevronDown.svg';
+import { PropsWithChildren } from 'react';
+
+interface Props {
+  isExpandedContent?: () => JSX.Element;
```

Comment:
now you can rename them to get/render*
as in `getExpandedContent` or `renderExpandedContent`

Reply:
done